checklist.md

```diff
@@ -1,206 +1,206 @@
-## Common mistakes (jv-lottery)
-
-#### Please don't add redundant empty lines to your code.
-We don't need them after class declaration or method signature.
-* Bad example:
-```
-public class Main {
```

Comment:
Please dont push ( and dont change if it is not necessary) files which is not related to homework